### PR TITLE
[DOC][Minor]Specify the common sources available for collecting

### PR DESCRIPTION
**Pull request by yjshen**
I was wondering what else common sources available until search the source code. Maybe better to make this clear.

### CODE REVIEW

File: conf/metrics.properties.template [hunk 1]
```diff
@@ -47,6 +47,10 @@
 #    instance master and applications. MetricsServlet may not be configured by self.
 #
 
+## List of available common sources and their properties.
+
+# org.apache.spark.metrics.source.JvmSource
```
Comment (srowen) [inline conf/metrics.properties.template]: This isn't the only `Source`, and this doesn't document anything about it. What's the value?

Comment (yijieshen) [inline conf/metrics.properties.template]: It's the only Common Source users can add additionally, all the spark internal source are added automatically. 
No value for this one, just as `org.apache.spark.metrics.sink.JmxSink`.

**Comment by srowen** [comment]
OK, if so then this seems at least harmless and has some value. LGTM.

**Comment by srowen** [comment]
OK to test

**Comment by srowen** [comment]
Jenkins, test this please

**Pull request closed without merging**